Norm_end_to_end.Rmd

```diff
@@ -77,70 +75,106 @@ naomit <- assay(res, "raw_filled") |>
     na.omit()
 ```
 
-### PCA unsupervised 
+### PCA unsupervised
 
 Look for overall biases in batch, sample, injection index,...
 
-```{r unsupervised checks, include=FALSE}
-pca_res <- prcomp(t(log2(naomit)), scale = FALSE, center = TRUE)
+```{r unsupervised checks, include=TRUE, fig.width = 10, fig.height = 5}
+## note: scale (center and scale) on columns (features after the t() call) to
+## remove dependency on absolute abundances.
+vals <- assay(res, "raw_filled_imputed") |>
+    log2() |>
+    t() |>
+    scale(center = TRUE, scale = TRUE)
+
+pca_res <- prcomp(vals, scale = FALSE, center = FALSE)
 
 par(mfrow = c(1, 2), mar = c(4.5, 4.5, 4.5, 1))
-plot_pca(pca_res, pc_x = 1, pc_y = 2, col = col_source)
-legend("topleft", inset = c(0, -0.24), col = col_source,
-       legend = names(col_source), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+plot_pca(pca_res, pc_x = 1, pc_y = 2, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
```

Comment:
for the *real* PCA we can then eventually remove this `labels = ...` - I just wanted to see if we have any dependency of the PCA on the injection index